openhands_cli/tui/tui.py

```diff
@@ -93,6 +94,25 @@ def display_welcome(conversation_id: UUID, resume: bool = False) -> None:
     """Display welcome message."""
     clear()
     display_banner(str(conversation_id), resume)
+
+    # Check for updates and display version info
+    version_info = check_for_updates()
+    print_formatted_text(
+        HTML(f"<grey>Version:</grey> <white>{version_info.current_version}</white>")
```

Comment:
@OpenHands start a new PR, we should make the current version display in the same grey color